x-pack/plugins/canvas/public/components/dom_preview/dom_preview.js

```diff
@@ -38,34 +38,34 @@ export class DomPreview extends React.Component {
   }
 
   update = original => () => {
-    const thumb = original.cloneNode(true);
+    if (this.content && this.container) {
```

Comment:
change this to an early return. Eg `if (!this.content || !this.container) return nullOrWhatever;`